custom_components/google_home/services.yaml

```diff
@@ -15,6 +15,12 @@ delete_alarm:
         entity:
           domain: sensor
           integration: google_home
+    skip_refresh:
+      example: false
+      default: true
```

Comment:
I meant default true for `force_refresh`. For `skip_refresh` it should be false.